lrauv_ignition_plugins/CMakeLists.txt

```diff
@@ -20,32 +20,32 @@ endif()
 
 #============================================================================
 # Find dependencies
-find_package(ignition-cmake3 REQUIRED)
+find_package(gz-cmake3 REQUIRED)
 
-find_package(ignition-gazebo7 REQUIRED COMPONENTS gui)
-set(IGN_GAZEBO_VER ${ignition-gazebo7_VERSION_MAJOR})
+find_package(gz-sim7 REQUIRED COMPONENTS gui)
+set(IGN_GAZEBO_VER ${gz-sim7_VERSION_MAJOR})
 
-find_package(ignition-gui7 REQUIRED)
-set(IGN_GUI_VER ${ignition-gui7_VERSION_MAJOR})
+find_package(gz-gui7 REQUIRED)
+set(IGN_GUI_VER ${gz-gui7_VERSION_MAJOR})
```

Comment:
@adityapande-1995, thanks for getting these updates in! Do you think we should update the `IGN_` globals to to `GZ_` as part of this update?

Reply:
Done : https://github.com/osrf/lrauv/pull/220/commits/e1960c833e647c0494178a63addf3cb159155b68